substrate/frame/nomination-pools/src/lib.rs

```diff
@@ -1598,7 +1577,7 @@ pub mod pallet {
 	use sp_runtime::Perbill;
 
 	/// The current storage version.
-	const STORAGE_VERSION: StorageVersion = StorageVersion::new(8);
+	const STORAGE_VERSION: StorageVersion = StorageVersion::new(9);
```

Comment:
Does this change the storage layout? If not, could name this migration just something like `fix_out_of_sync_tvl` without needing to bump this

Reply:
Done, thanks!